src/VolleyManagement.Backend/VolleyManagement.Data.MsSql/Queries/GameResultQueries.cs

```diff
@@ -37,11 +32,7 @@ public class GameResultQueries : IQuery<GameResultDto, FindByIdCriteria>,
         /// <param name="unitOfWork">Instance of class which implements <see cref="IUnitOfWork"/>.</param>
         public GameResultQueries(IUnitOfWork unitOfWork)
         {
-            _unitOfWork = (VolleyUnitOfWork)unitOfWork;
```

Comment:
In order to get rid of unnecessary dependencies.
Also because they represented the same functionality, which could be done by a single UoW instance. And it works well.

![default](https://user-images.githubusercontent.com/11539042/36491541-a6641948-1733-11e8-9db5-69dc83ecc410.PNG)

Equals.

![2](https://user-images.githubusercontent.com/11539042/36491627-e473c7c4-1733-11e8-9c2b-4e6d9090f773.PNG)

Reply:
UoW already contains Context with dbSets.

Reply:
You think so because you never wrote those queries. After you type *_unitOfWork.Context.* for the 100th time then you will get this variables back.

So choice is yours:
1. you can believe me and revert those changes
2. I can give you a query writing practice - and you will ask me to revert those changes back

What do you wish to do?

Reply:
So I have just an illusion of choice - it will be reverted. Ok, than my ecision falls for  #1...but I still don't get it.